services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/HankeFactory.kt

```diff
@@ -99,6 +100,12 @@ class HankeFactory(
                 hankeStatus,
             )
 
+        /** Create minimal Entity with identifier fields and mandatory fields. */
+        fun createNewEntity(
```

Comment:
The name could tell the difference between this and `create()`.
```suggestion
        fun createMinimal(
```

Reply:
Fixed.